openmc/statepoint.py

```diff
@@ -312,6 +316,13 @@ def run_mode(self):
         return self._f['run_mode'].value.decode()
 
     @property
+    def runtime(self):
+        out = dict()
+        for key in self._f['runtime'].keys():
+            out[key] = self._f['runtime/' + key].value
+        return out
```

Comment:
Could do a one-liner here:
`return {name: dataset.value for name, dataset in self._['runtime'].items()}`